### PR TITLE
Chore(Enhancement) - Add a command that cleans up compiled files.

### LINKED ISSUE #144: [Suggestion] Clear existing *rpyc button
Dunno if it's in the scope of what this plugin is going for but an easy button to clear out old rpyc files since they're hidden by default would be nice. In bash its as simple as running `find . -type f -name '*.rpyc' -delete`, which on one hand makes it kinda redundant but hey, why not?

### PR DESCRIPTION
**Pull request by a2937**
## Description
It cleans up a few minor misspellings and properly adds a command to move all compiled files that don't have source files to the old-game directory. The command heading is "renpy.migrateOldFiles" and the formal title is "Cleanup unused compiled Ren'Py source files" 

## Related Issue

Closes #144

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +84,10 @@
         "command": "renpy.refreshDiagnostics",
         "title": "Refresh Ren'Py diagnostics for the active editor window"
       },
+      {
+        "command": "renpy.migrateOldFiles",
+        "title": "Cleanup unused compiled Ren'Py source files"
```
Comment (duckdoom4) [inline package.json:89]: I just pushed a commit on the main branch that updates these to match the convention. That is; to add an identifier before the command. Could you update this to "Renpy: Cleanup ..."

**Review by duckdoom4 — CHANGES_REQUESTED** [review]
Bit of a nitpick maybe, but I'd prefer the use of await instead of then.
That reduces indentation and will imho make it a lot more readable.

File: package.json (hunk 1 above)
Comment (a2937) [reply package.json]: Okay done.

**Review by duckdoom4 — APPROVED** [review]
Awesome work! LGTM!

**Review by LuqueDaniel — APPROVED** [review]
LGTM!

**Pull request merged**